src/Akka.Logger.Serilog/SerilogLogMessageFormatter.cs

```diff
@@ -37,6 +38,9 @@ public SerilogLogMessageFormatter()
         /// </returns>
         public string Format(string format, params object[] args)
         {
+            if (Log.Logger.BindMessageTemplate(format, args, out var boundTemplate, out var boundProps))
```

Comment:
Please add a comment explaining what this does and how the fallback mechanism works